### DISCUSSION TITLE
Technical Design of the Testimonial Free Website

### DISCUSSION THREAD

**Discussion by vinodsharma10x** [careerJumpstart/TestimonialFree#6] [General]
Read at https://github.com/careerJumpstart/TestimonialFree/wiki and discuss here

**Comment by roberto-morado** [comment]
I consider the current state of the file directory to be redundant, I also noticed some useless files being pushed to the repository; I'd suggest updating the `gitignore` file and to come to a common agreement on how to organize the codebase; e.g. 

|
| 📂 docs
| - 📄 DEVPLAN
| 📂 frontend
| - 📄 package.json
| - 📄 ...
| - 📁 ...
| 📁 backend
| 📄 README
| 📄 LICENSE
|

See c95e920b4ba75f4beb3f1018fcb43b9af92d24f9
